docs/specs/validation/heading-validation.yml

```diff
@@ -1323,6 +1323,44 @@ outputs:
     {"message_severity":"suggestion","log_item_type":"user","code":"duplicate-h1","message":"H1 'Title 1' is duplicated with other articles: 'a.md, b.md'","file":"b.md","line":1,"end_line":1,"column":1,"end_column":1}
     {"message_severity":"suggestion","log_item_type":"user","code":"duplicate-h1","message":"H1 'Title 1' is duplicated with other articles: 'a.md, b.md'","file":"a.md","line":1,"end_line":1,"column":1,"end_column":1}
 ---
+# Content validation - duplicated h2
```

Comment:
nit: a simpler case would be to assert the title of

```
# `<`
```

is `<` in _markdown.yml_

Reply:
added